src/pages/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,36 @@
+import "./UserProfile.css";
+import { Sidebar } from "components";
+import { useAuth } from "customHooks";
+
+const UserProfile = () => {
+  const { logoutHandler } = useAuth();
+  const user = JSON.parse(localStorage.getItem("VL_user"));
+
+  const fullName = `${user.firstName} ${user.lastName}`;
+
+  return (
+    <section className="main-section">
+      <Sidebar />
+
+      <div className="component-container profile-container">
+        <div className="profile-details">
+          <img
+            src={`https://avatars.dicebear.com/api/initials/${user.firstName}%20${user.lastName}.svg?b=%238553fa&r=50&size=80&fontSize=40`}
+            alt={fullName}
```

Comment:
you can extract this into utils so that the code becomes clean!

Reply:
Okay, I'll do that. Thanks.